backend/recommendation-processor/src/service/instagram-api/instagram-api-parser.ts

```diff
@@ -33,4 +50,39 @@ const getInstagramProfileDetails = async (instagramId: string): Promise<Instagra
 
     const response: AxiosResponse = await axios.request(options);
     return response.data.data as InstagramApiResponse
-}
\ No newline at end of file
+}
+
+const generateGoogleApiRequestBody = (instagramTableRecord: InstagramTable): GoogleApiPlaceIdSearchBody => {
+    const textQuery = getTextQuery(instagramTableRecord)
+    const locationBias = getLocationBias(instagramTableRecord)
```

Comment:
If no longitude and latitude are found in **instagramTableRecord** -- return undefined from this method. 
We consider no specified location on Instagram as the desired behaviour. 
No location -- no unifying